backend/src/main/java/gov/cdc/usds/simplereport/api/devicetype/DeviceTypeMutationResolver.java

```diff
@@ -27,4 +28,20 @@ public DeviceType updateDeviceType(@Argument UpdateDeviceType input)
       throws IllegalGraphqlArgumentException {
     return deviceTypeService.updateDeviceType(input);
   }
+
+  @MutationMapping
+  public DeviceType markDeviceTypeAsDeleted(@Argument UUID deviceId, @Argument String deviceName)
+      throws IllegalGraphqlArgumentException {
+    DeviceType device =
+        (deviceId != null)
+            ? deviceTypeService.getDeviceType(deviceId)
+            : deviceTypeService.getDeviceType(deviceName);
+
+    if (device == null) {
+      return null;
+    }
```

Comment:
I tested this locally, and I don't think this null check is necessary! `removeDeviceType` does nothing if the value is null.

Reply:
Nice! I removed the conditional but left the unit test for the scenario